bugbot/rules/component.py

```diff
@@ -89,6 +89,55 @@ def get_bugs(self, date="today", bug_ids=[]):
         # Classify those bugs
         bugs = get_bug_ids_classification("component", bug_ids)
 
+        # Collect bugs classified as Fenix::General
+        fenix_general_bug_ids = [
+            bug_id
+            for bug_id, bug_data in bugs.items()
+            if bug_data.get("class") == "Fenix"
+            and bug_data["prob"][bug_data["index"]]
+            >= self.get_config("general_confidence_threshold")
+        ]
+
+        def get_confidence_threshold(
+            bug_data, general_confidence_threshold, confidence_threshold
+        ):
+            if bug_data["class"] == "General":
+                return general_confidence_threshold
+            return confidence_threshold
+
+        # Collection bugs that were originally Fenix::General but reclassified to another product with low confidence
+        originally_fenix_general_bug_ids = [
+            bug_id
+            for bug_id, bug_data in bugs.items()
+            if raw_bugs[bug_id]["product"] == "Fenix"
+            and raw_bugs[bug_id]["component"] == "General"
+            and bug_data["prob"][bug_data["index"]]
+            <= get_confidence_threshold(
+                bug_data,
+                self.get_config("general_confidence_threshold"),
+                self.get_config("confidence_threshold"),
+            )
+        ]
+
+        fenix_general_bug_ids.extend(originally_fenix_general_bug_ids)
+        fenix_general_bug_ids = set(fenix_general_bug_ids)
```

Comment:
We could have it as set from the beginning instead of converting it to set later.

Reply:
Done in bc68d06132884591efb2653977567238bd0ea681!